hypha/static_src/sass/components/_wrapper.scss

```diff
@@ -127,6 +127,13 @@
         }
     }
 
+    // flex
+    &--flex {
+        display: flex;
+        align-items: start;
+        column-gap: 25px;
```

Comment:
A rem value is preferred. Just "gap" works as well unless rows should not have gaps.